api/v1alpha1/gatewayclass_types.go

```diff
@@ -26,22 +26,95 @@ import (
 // +k8s:openapi-gen=true
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// GatewayClass describes a class of gateways av
-// for defining access to their routed services. GatewayClass allow a
+// GatewayClass describes a class of gateways available to the user
+// for creating Gateway resources.
 //
-// GatewayClass is a non-namespaced resource.
+// GatewayClass is a Cluster level resource.
+//
+// Support: Core.
```

Comment:
@bowei should we have a standard way to represent the support-level of API fields, i.e. kubebuilder markers? For example:
```
// +serviceapis:resource:core
// +serviceapis:resource:extended
// +serviceapis:resource:custom
```

Future tools may be able to use these markers in interesting ways.

Reply:
Sounds good -- is this something more standardized that we can refer to? I guess the kubebuilder parser already understand this style of annotation.

Reply:
I'm going to file an issue covering this, then we can make it consistent everywhere in the codebase in one fell swoop:

https://github.com/kubernetes-sigs/service-apis/issues/56